Document/0x06h-Testing-Platform-Interaction.md

```diff
@@ -23,9 +23,17 @@ $ strings <yourapp> | grep "myURLscheme://"
 
 #### Dynamic Analysis
 
-Once you know, what an URL structure is, you should try fuzzing an URL to force an application to perform some malicious action.
+As custom URL schemes may contain bugs, it is worth fuzzing those URLs. To do it you may use [IDB](http://www.idbtool.com/) tool:
 
--- TODO [Add instruction of using dynamic/ipc/open_uri: Test IPC attacks by launching URI Handlers in Needle: https://labs.mwrinfosecurity.com/blog/needle-how-to/] --
+- Connect IDB tool with your device and select tested application. [Here](http://www.idbtool.com/documentation/setup.html) you can find a detailed guide in setting it up.
```

Comment:
This should be something like: "The `[`IDB documentation`]` has instructions...", so that it still works if the document is printed.